<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Users</class>
 <widget class="QDialog" name="Users">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>LingWai SC</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_login">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>350</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Baskerville</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Sign in</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_back1" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>271</width>
     <height>601</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>151</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Baskerville</family>
      <pointsize>50</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>Log In</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_back">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>410</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Baskerville</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_email">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>160</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Baskerville</family>
    </font>
   </property>
   <property name="text">
    <string>EMAIL</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_password">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>220</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Baskerville</family>
    </font>
   </property>
   <property name="text">
    <string>PASSWORD</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_email">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>180</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_password">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>240</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>280</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Baskerville</family>
    </font>
   </property>
   <property name="text">
    <string>没有用户，创建一个？</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="commandLinkButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>300</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Baskerville</family>
     <pointsize>13</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>创建账号</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
